hedera-mirror-rest/api/v1/openapi.yml

```
@@ -983,6 +983,29 @@ components:
         tokens:
           - token_id: 0.0.200001
             balance: 8
+    ChunkInfo:
+      type: object
+      properties:
+        number:
+          type: integer
+        total:
+          type: integer
+        initial_transaction_id:
+          type: string
+        nonce:
+          type: integer
+        scheduled:
+          type: boolean
+      required:
+        - number
+        - total
+        - initial_transaction_id
```

remove the required section.

```suggestion
      required:
        - number
        - total
        - initial_transaction_id
```

---

Removed